packages/aws-cdk/test/integ/cli/test-cdk-version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+source ${scriptdir}/common.bash
+# ----------------------------------------------------------
+
+setup
+
+assert "cdk version" "cdk --version"
```

Comment:
Did you ever run this test successfully? This is not the API for `assert`, which compares the result of the first command with STDIN (it doesn't take 2 arguments).